GitCommands/Repository/RecentRepoInfo.cs

```diff
@@ -68,7 +68,7 @@ public class RecentRepoSplitter
 
         public RecentRepoSplitter()
         {
-            MaxRecentRepositories = AppSettings.MaxMostRecentRepositories;
```

Comment:
MaxMostRecentRepositories should be untouched, you have to introduce a new setting for RecentRepositoryHistorySize

Reply:
reworked